samtools.1

```diff
@@ -88,6 +88,8 @@ samtools addreplacerg -r 'ID:fish' -r 'LB:1334' -r 'SM:alpha' -o output.bam inpu
 samtools collate aln.sorted.bam aln.name_collated.bam
 .PP
 samtools depad input.bam
+.PP
+samtools markdup in.algnsorted.bam out.bam
```

Comment:
From the man page: 

` Mark duplicate alignments from a coordinate sorted file that
  has been run through fixmate with the -m option.
`